document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/service/AccountsService.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.companieshouse.document.generator.accounts.service;
+
+import uk.gov.companieshouse.api.model.accounts.Accounts;
+
+public interface AccountsService {
+
+    /**
+     * Call the api sdk to get accounts data of resource passed.
+     *
+     * __NOTE:__: It is important to note, currently it will call an internal accounts manager
+     * within this project that calls the abridged api through ERIC as the private sdk hasn't
+     * been built yet. When the private SDK has been built, we should be able to simply delete the
+     * accounts package and use the SDK in the implementation of this method.
```

Comment:
Can this be linked to the task to implement the private SDK please. This will allow developers reading this to be able to easily check if the dependent work is completed.